src/store/printer/getters.ts

```diff
@@ -101,6 +101,19 @@ export const getters: GetterTree<PrinterState, RootState> = {
   },
 
   getPrintProgress: (state) => {
+    const { gcode_start_byte, gcode_end_byte, filename } = state.printer.current_file ?? {}
+    const { file_position } = state.printer.virtual_sdcard ?? {}
+
+    if (gcode_start_byte && gcode_end_byte && file_position && filename === state.printer.print_stats?.filename) {
```

Comment:
`filename === state.printer.print_stats?.filename`

With the `const` on line 104 defining `filename` in the scope of this method, and the way this `filename` is being defined inline creates an in-scope variable of, well, `filename`.

However, this variable is not used anywhere in this scope below, between lines 108 to 114.

Do we need this check?

Reply:
The `filename` on 104 is from `state.printer.current_file`, the check on 107 is with `state.printer.print_stats` (different parent objects).

This is a sanity check as the 2 parent objects can be updated at different times, making them out-of-sync.